setup.py

```diff
@@ -12,7 +12,7 @@
     long_description_content_type="text/markdown",
     package_dir={"": "src"},
     packages=setuptools.find_packages("src", exclude=["tests"]),
-    install_requires=["rstr", "faker", "smart_open", "jsonschema", "typer", "pydantic"],
+    install_requires=["rstr", "faker", "smart_open", "jsonschema", "typer", "pydantic", "defusedxml", "requests", "json2xml"],
```

Comment:
Since XML wont be added in the package as noted below, could you remove this?

Reply:
Does not work for me for some reason.

Reply:
I am on ubuntu.
command used: `pytest`

pwd: jsf/src

lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 20.04.3 LTS
Release:        20.04
Codename:       focal

Reply:
The above comments are supposed to be for pytest failure I mentioned above.